core/lib/spree/testing_support/i18n.rb

```diff
@@ -84,7 +84,7 @@ def self.translations
       end
 
       Spree.check_unused_translations
-      if false && Spree.unused_translation_messages.any?
+      if Spree.unused_translation_messages.any?
```

Comment:
I've removed the `false` keyword as DeepSource raised `Literal being used as a condition error. Also, `false && something` effectively stops the following block from getting executed ever. And given that the below `puts` blocks appear to be logging messages when the `Spree.unused_translation_messages` array is not empty, I went ahead and removed the `false` condition.